Dockerfile

```diff
@@ -76,6 +76,11 @@ COPY root/etc/cont-init.d/60-firefox.sh /etc/cont-init.d/
 
 ADD ./src/common/xfce/ /config
 
+# Terra customizations here!!
+ADD default /etc/nginx/sites-available/default
+ADD server.xml /usr/local/tomcat/conf/server.xml
+ENV RSTUDIO_HOME "unused"
```

Comment:
This environment variable is read by Terra - in the future we will remove the need for this.